src/bun.js/bindings/exports.zig

```diff
@@ -1839,6 +1832,26 @@ pub const ZigConsoleClient = struct {
                     this.formatter.writeIndent(Writer, this.writer) catch {};
                 }
 
+                fn getObjectName(this: *@This(), globalThis: *JSC.JSGlobalObject, value: JSValue) ZigString {
```

Comment:
remove unused parameter `this`. it will make it so you call this fucntion just as `getObjectName(...)` and not `this.getObjectName(...)`

Reply:
Good point. Done.